code/game/objects/items/explosives/mine.dm

```diff
@@ -264,6 +274,15 @@
 	if(!QDELETED(src))
 		disarm()
 
+/obj/item/explosive/mine/bullet_act(obj/projectile/xenoprojectile)
```

Comment:
xeno_projectile

we use a formatting called snake case where words are all lowercase separated by underscores (can see it there with bullet_act)